.github/workflows/linux.yml

```diff
@@ -358,7 +358,7 @@ jobs:
         run: |
           source ${OV_INSTALL_DIR}/setupvars.sh
           python -m pip install . --verbose --find-links ${OV_INSTALL_DIR}/wheels
-          python -m pytest -v ./tests/python_tests/test_whisper_generate_api.py -k "not test_smoke"
+          python -m pytest -v ./tests/python_tests/test_whisper_generate_api.py ./tests/python_tests/test_whisper_pipeline_static.py -k "not test_smoke"
```

Comment:
Should `./tests/python_tests/test_whisper_pipeline_static.py` be on separate line?

Reply:
can be, but it works like this (we just list all test files to run here).